src/test/TestGhoBase.t.sol

```diff
@@ -292,20 +311,137 @@ contract TestGhoBase is Test, Constants, Events {
     );
-    GHO_TOKEN.grantRole(GHO_TOKEN_BUCKET_MANAGER_ROLE, address(GHO_STEWARD_V2));
-    GHO_GSM.grantRole(GSM_CONFIGURATOR_ROLE, address(GHO_STEWARD_V2));
+
+    // Deploy Gho CCIP Steward
+    GHO_CCIP_STEWARD = new GhoCcipSteward(
+      SHORT_EXECUTOR,
+      address(GHO_TOKEN),
+      address(GHO_TOKEN_POOL),
+      RISK_COUNCIL
+    );
+
+    // Deploy Gho GSM Steward
+    GHO_GSM_STEWARD = new GhoGsmSteward(SHORT_EXECUTOR, RISK_COUNCIL);
+
+    // Deploy Bucket Capacity Manager
+    BUCKET_CAPACITY_MANAGER = new BucketCapacityManager(
+      SHORT_EXECUTOR,
+      address(GHO_TOKEN),
+      RISK_COUNCIL
+    );
     address[] memory controlledFacilitators = new address[](2);
     controlledFacilitators[0] = address(GHO_ATOKEN);
     controlledFacilitators[1] = address(GHO_GSM);
     vm.prank(SHORT_EXECUTOR);
-    GHO_STEWARD_V2.setControlledFacilitator(controlledFacilitators, true);
+    BUCKET_CAPACITY_MANAGER.setControlledFacilitator(controlledFacilitators, true);
+
+    // Setup GHO Token Pool
+    uint64 SOURCE_CHAIN_SELECTOR = 1;
+    uint64 DEST_CHAIN_SELECTOR = 2;
+    UpgradeableTokenPool.ChainUpdate[] memory chainUpdate = new UpgradeableTokenPool.ChainUpdate[](
+      1
+    );
+    chainUpdate[0] = UpgradeableTokenPool.ChainUpdate({
+      remoteChainSelector: DEST_CHAIN_SELECTOR,
+      allowed: true,
+      outboundRateLimiterConfig: getOutboundRateLimiterConfig(),
+      inboundRateLimiterConfig: getInboundRateLimiterConfig()
+    });
+    vm.prank(OWNER);
+    GHO_TOKEN_POOL.applyChainUpdates(chainUpdate);
+
+    // Deploy Arb Gho Token pool
+    UpgradeableBurnMintTokenPool arbTokenPoolImpl = new UpgradeableBurnMintTokenPool(
+      address(GHO_TOKEN),
+      ARM_PROXY,
+      false
+    );
+    tokenPoolInitParams = abi.encodeWithSignature(
+      'initialize(address,address[],address)',
+      OWNER,
+      emptyArray,
+      ROUTER
+    );
+    // proxy deploy and init
+    tokenPoolProxy = new TransparentUpgradeableProxy(
+      address(arbTokenPoolImpl),
+      PROXY_ADMIN,
+      tokenPoolInitParams
+    );
+    // Manage ownership
+    vm.prank(OWNER);
+    UpgradeableBurnMintTokenPool(address(tokenPoolProxy)).acceptOwnership();
+    ARB_GHO_TOKEN_POOL = UpgradeableBurnMintTokenPool(address(tokenPoolProxy));
+
+    // Setup Arb GHO Token Pool
+    vm.prank(OWNER);
+    ARB_GHO_TOKEN_POOL.applyChainUpdates(chainUpdate);
+
+    // Deploy Arb Gho CCIP Steward
+    ARB_GHO_CCIP_STEWARD = new GhoCcipSteward(
+      SHORT_EXECUTOR,
+      address(GHO_TOKEN),
+      address(ARB_GHO_TOKEN_POOL),
+      RISK_COUNCIL
+    );
+
+    // Deploy Arb Bucket Capacity Manager
+    ARB_BUCKET_CAPACITY_MANAGER = new BucketCapacityManager(
+      SHORT_EXECUTOR,
+      address(GHO_TOKEN),
+      RISK_COUNCIL
+    );
+    vm.prank(SHORT_EXECUTOR);
+    ARB_BUCKET_CAPACITY_MANAGER.setControlledFacilitator(controlledFacilitators, true);
+  }
+
+  function getOutboundRateLimiterConfig() public pure returns (RateLimiter.Config memory) {
```

Comment:
why do we need this getter?

Reply:
See if they're used, if as constants, change name

Reply:
Done